.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: Compile the protocol buffers, launch the backend with docker-compose
+    before: cd app
+    init: |
+      docker run --rm -w /app -v $(pwd):/app couchers/grpc ./generate_protos.sh
+      docker-compose -f docker-compose.yml -f docker-compose.gitpod.yml up --build
+
+  - name: Install and launch the frontend with yarn
+    before: cd app/frontend
+    init: |
+      yarn install
+      gp await-port 8888
```

Comment:
Is this needed anymore? It should be able to `yarn start` before the backend is up?

Reply:
Yeah, but the build is usually a lot slower than the frontend dev server so if the preview is up and people click around at that in between point, it'll error until the backend finish building, so I thought it might be better to wait for it to be up first before starting the frontend

Reply:
Ahh, fair enough. It just adds a bit, I guess since it'll only actually build the frontend when you do `yarn start`?

Reply:
But yeah; I don't mind leaving it in!